### PR TITLE
MGMT-17150 MGMT-16852 Merge Devices and ER pages

### PR DESCRIPTION
**Pull request by rawagner**
Add download qcow2 or bootc commands

add_device_bootc
add_device_qcow

### CODE REVIEW

**Comment by fzdarsky** [comment]
We cannot onboard devices into the Device list. They *have to* become part of the Enrollment Request list first, then be transferred to the Device list.

File: webpack.config.ts [hunk 1]
```diff
@@ -192,6 +192,8 @@ if (NODE_ENV === 'production') {
       'window.KEYCLOAK_CLIENTID': JSON.stringify(process.env.KEYCLOAK_CLIENTID),
       'window.KEYCLOAK_REDIRECT': JSON.stringify(process.env.KEYCLOAK_REDIRECT),
       'window.API_PORT': JSON.stringify(process.env.API_PORT) || '3001',
+      'window.QCOW2_IMG_URL': JSON.stringify(process.env.QCOW2_IMG_URL),
+      'window.BOOTC_IMG_URL': JSON.stringify(process.env.BOOTC_IMG_URL),
```
[the remark this answers is not in the document]
Comment (rawagner) [reply webpack.config.ts:196]: yes, once we merge, we will need to add values for these env vars into deployment

[the remark this answers is not in the document]
Comment (rawagner) [reply webpack.config.ts:196]: I misunderstood. I've added the variables to the deployment files now.

File: src/app/components/Device/DeviceList.tsx [hunk 2]
```diff
@@ -1,98 +1,338 @@
...
-const DeviceListTable = () => {
-  const [devicesList, loading, error, refetch] = useFetchPeriodically<DeviceList>({
+const DeviceList = () => {
+  const [addDeviceModal, setAddDeviceModal] = React.useState(false);
+  const [devicesList, loading, error, devicesRefetch] = useFetchPeriodically<DeviceList>({
     endpoint: 'devices',
   });
 
+  const [erList, erLoading, erError, erRefetch] = useFetchPeriodically<DeviceList>({
+    endpoint: 'enrollmentrequests',
```
[the remark this answers is not in the document]
Comment (rawagner) [reply src/app/components/Device/DeviceList.tsx:135]: We have a discussion planned to see if we will want to merge these two into one table. If we agree, I will create a ticket for BE.

**Review by celdrake — APPROVED** [review]

**Pull request merged**